pom.xml

```diff
@@ -98,7 +98,7 @@
         <dependency>
             <groupId>com.azure</groupId>
             <artifactId>azure-monitor-opentelemetry-exporter</artifactId>
```

Comment:
Have you try to make a native build?

Is this mandatory `<arg>--initialize-at-build-time=com.azure.core.util.logging.ClientLogger</arg>` like sender-auth?

Reply:
This microservice doesn't have native build yet.
The relative task has been created, thank you!